src/test/java/net/openhft/chronicle/queue/RollCyclesDistinctnessTest.java

```diff
@@ -0,0 +1,22 @@
+package net.openhft.chronicle.queue;
+
+import org.junit.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+
+public class RollCyclesDistinctnessTest {
+    
+    @Test
+    public void definedRollCycleFormatsAreDistinct() {
```

Comment:
Add a test to ensure the format string is distinct for all our enumerated RollCycles